packages/at_secondary_server/bin/main.dart

```diff
@@ -9,7 +9,7 @@ import 'package:at_utils/at_logger.dart';
 /// given port.
 /// Throws [SocketException] on invalid port.
 /// Throws [ArgParserException] for invalid arguments passed.
-/// @ param - List<String> atSign and port
+/// @ param - List&lt;String&gt; atSign and port
```

Comment:
lint

Reply:
That's not really helpful for readability.

Reply:
Is it because we don't have a ` around the code fragment. I see some of the later lints left < and > inside code fences alone.